css/properties/position-try-fallbacks.json

```
@@ -749,11 +749,25 @@
         },
         "self-x-end": {
           "__compat": {
-            "spec_url": "https://drafts.csswg.org/css-anchor-position-1/#valdef-position-area-self-x-end",
+            "spec_url": "https://drafts.csswg.org/css-anchor-position/#valdef-position-area-self-x-end",
```

Nit: I think it's preferable to include the level in the URL.


```suggestion
            "spec_url": "https://drafts.csswg.org/css-anchor-position-1/#valdef-position-area-self-x-end",
```